src/box/memtx_tree.cc

```diff
@@ -1780,7 +1782,8 @@ static const struct index_vtab memtx_tree_use_hint_index_vtab = {
 	/* .max = */ generic_index_max,
 	/* .random = */ memtx_tree_index_random<true>,
 	/* .count = */ memtx_tree_index_count<true>,
-	/* .get = */ memtx_tree_index_get<true>,
+	/* .get_raw = */ memtx_tree_index_get_raw<true>,
+	/* .get = */ memtx_index_get,
```

Comment:
I don't like additional redirection. Could you consider to use my patch alyapunov/gh-2695-index-refactoring? The you can add another template argument `bool COMPRESSED` and use `memtx_index_get` only if not.